### PR TITLE
Add command-line option to list the supported licenses

### PR DESCRIPTION
**Pull request by Hoverth**
Does what it says on the tin. Did have to update `structopt` to the latest version though, just to be transparent.

You can now run `license-generator --list` and it will simply print out all the licenses given in the LICENSES const variable.

If you pass `--list` it doesn't require an INPUT set, but if `--list` isn't there, you do

### CODE REVIEW

**Comment by Hoverth** [comment]
I do not know why it has my previous commits here, and I do not know how to be rid of them.

File: src/main.rs [hunk 1]
```diff
@@ -25,6 +27,19 @@ struct Opt {
...
+        println!("Supported Licenses are:");
+        use license_generator::license::LICENSES;
+        for l in LICENSES {
+            print!("{}, ", l);
+        }
+        println!();
+        return Ok(());
+    } else {
+        
+    }
```
Comment (azu) [inline src/main.rs:41]: Is this `else` clause necessary?

Comment (Hoverth) [reply src/main.rs]: Ah, for goodness’ sake. I did fix this on my local repo, but git was not liking me yesterday. No, that else clause is very much not necessary.

**Comment by Hoverth** [comment]
That should fix the random else statement.

**Review by azu — COMMENTED** [review]
Almost LGTM.

Can you update README too?
https://github.com/azu/license-generator/blob/master/README.md?plain=1#L11-L34

**Comment by Hoverth** [comment]
Is that all good?

**Review by azu — APPROVED** [review]
LGTM

**Comment by azu** [comment]
> I do not know why it has my previous commits here, and I do not know how to be rid of them.

Probably, it is safe that git fetch upstream repo(it is `https://github.com/azu/license-generator`) and checkout any feature branch before submitting PR.

- [git - After rebasing my github fork, commits from others are in my pull request - Stack Overflow](https://stackoverflow.com/questions/40413071/after-rebasing-my-github-fork-commits-from-others-are-in-my-pull-request)
- [Syncing a fork - GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork)

**Comment by azu** [comment]
Release [Release v1.3.0 · azu/license-generator](https://github.com/azu/license-generator/releases/tag/v1.3.0) 🎉

**Pull request merged**